feature-staff/src/main/java/io/github/droidkaigi/confsched2022/feature/staff/Staff.kt

```diff
@@ -0,0 +1,102 @@
+package io.github.droidkaigi.confsched2022.feature.staff
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import coil.compose.AsyncImage
+import io.github.droidkaigi.confsched2022.designsystem.theme.KaigiScaffold
+import io.github.droidkaigi.confsched2022.designsystem.theme.KaigiTopAppBar
+import io.github.droidkaigi.confsched2022.feature.staff.StaffUiModel.StaffState.Loaded
+
+@Composable
+fun StaffScreenRoot(
+    modifier: Modifier = Modifier,
+    showNavigationIcon: Boolean = true,
+    onNavigationIconClick: () -> Unit = {}
+) {
+    val viewModel = hiltViewModel<StaffViewModel>()
+    val uiModel by viewModel.uiModel
+    Staff(uiModel, showNavigationIcon, onNavigationIconClick, modifier)
+}
+
+@Composable
+fun Staff(
+    uiModel: StaffUiModel,
+    showNavigationIcon: Boolean,
+    onNavigationIconClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    KaigiScaffold(
+        topBar = {
+            KaigiTopAppBar(
+                showNavigationIcon = showNavigationIcon,
+                onNavigationIconClick = onNavigationIconClick,
+                title = {
+                    Text(
+                        text = stringResource(id = R.string.staff_top_app_bar_title),
+                        style = MaterialTheme.typography.titleLarge
+                    )
+                },
+            )
+        }
+    ) {
+        if (uiModel.staffState !is Loaded) {
+            CircularProgressIndicator()
+            return@KaigiScaffold
+        }
+        val staff = uiModel.staffState.staff
+
+        LazyColumn(
+            modifier = modifier.fillMaxWidth()
+        ) {
+            items(items = staff, key = { it.id }) { staff ->
+                Row(
```

Comment:
This item is the same as the contributor [row](https://github.com/DroidKaigi/conference-app-2022/blob/main/feature-contributors/src/main/java/io/github/droidkaigi/confsched2022/feature/contributors/Contributors.kt#L87-L114). 

Do you think we could share the UiComponent?

Reply:
@kittinunf 
Thanks for the great point out. 
I have a consultation.

I think this refactoring should add modules to put common UI components.
If I respond with this Pull Request, the change will be large, so is it okay to create a new issue?
Like a  "Add a module for placing common UI parts, such as the `common-ui-component` module"

I'll wait and see and if no one is assigned, I'll raise my hand.

Reply:
I feel like core-ui might be a good candidate for this, it should be a foundation module for all of the feature modules. What do you think?

cc: @takahirom

Reply:
How about having it here?
https://github.com/DroidKaigi/conference-app-2022/tree/main/core-designsystem/src/androidMain/kotlin/io/github/droidkaigi/confsched2022/designsystem/components

Reply:
Thanks, I had overlooked core-designsystem module.
looks good.

Sorry for the delay, I’ll fix it at a convenient time today.

Reply:
@takahirom 
I created the rows used in the staff list and contributors as a common component.
[89af427](https://github.com/DroidKaigi/conference-app-2022/pull/331/commits/89af427d922ffeeff0e0810091218c47ecc1d2e5)